modules/cudf/test/cudf-data-frame-tests.ts

```diff
@@ -437,3 +438,19 @@ test('dataframe.nansToNulls', () => {
 
   expect(result.get('b').nullCount).toEqual(1);
 });
+
+test('dataframe.isNull', () => {
+  const a      = Series.new({type: new Int32, data: [0, null, 2, 3, null]});
+  const b      = Series.new({type: new Float32, data: new Float32Buffer([NaN, 0, 3, NaN, 5])});
+  const c      = StringSeries.new([null, null, 'foo', 'bar', '']);
```

Comment:
Why not use `Series.new()`?
```suggestion
  const c      = Series.new([null, null, 'foo', 'bar', '']);
```